server/graph.go

```
@@ -28,5 +29,6 @@ func liveGDM(sr graph.StateReader) (*LiveGDM, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &LiveGDM{gdm.Deployments}, nil
+	etag, _ := state.GetEtag()
```

Should prob not swallow this error, could be confusing.